testing/fs-uae-launcher/APKBUILD

```diff
@@ -1,20 +1,18 @@
 # Maintainer: Taner Tas <taner76@gmail.com>
 pkgname=fs-uae-launcher
-pkgver=2.8.4
+pkgver=3.0.0
 pkgrel=0
 pkgdesc="FS-UAE emulator launcher."
 url="https://fs-uae.net/launcher"
 arch="noarch !armhf !armv7 !aarch64 !s390x"  # fs-uae not avail
 license="GPL-2.0-only"
-depends="fs-uae~$pkgver py3-qt5"
+depends="py3-qt5 py3-sip-pyqt5"
```

Comment:
I don't know the package, but does the launcher really not need the actual emulator?

Reply:
It causes other problems during package build. This is better in my opinion. If you think otherwise, feel free to add it for yourself.

Reply:
define "other problems" please.

Reply:
It couldn't detect proper version of the fs-uae thus unable to build the package. This shouldn't be happen since fs-uae is not a build dependency.

Is this definition enough? Or should I file a full report about the problem sir?

Reply:
Just depend on fs-uae=$pkgver and it should work.

Reply:
Ok. Done.